From a088f5cce9884294ac29f04f57e5842c5c16b28f Mon Sep 17 00:00:00 2001
From: Vishesh Handa <me@vhanda.in>
Date: Sat, 29 Mar 2014 19:06:54 +0100
Subject: [PATCH 23/56] BasicIndexingJob: Do not add the type as 'file'

Everything in the "file database" is a file. We're no longer in the
Nepomuk world where the same db contains all the different resources.
---
 src/file/basicindexingjob.cpp | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/file/basicindexingjob.cpp b/src/file/basicindexingjob.cpp
index dfd9fb4..4bbcc55 100644
--- a/src/file/basicindexingjob.cpp
+++ b/src/file/basicindexingjob.cpp
@@ -81,7 +81,6 @@ bool BasicIndexingJob::index()
         QString tstr = KFileMetaData::TypeInfo(type).name().toLower();
         doc.addBoolTerm(tstr, "T");
     }
-    doc.addBoolTerm("file", "T");
 
     if (fileInfo.isDir()) {
         doc.addBoolTerm("folder", "T");
-- 
1.9.1

